Accessories/ExpansionBoard_uInverter/docs/uInverter_connections.md

```diff
@@ -0,0 +1,45 @@
+# uInverter Board
+
+This document decribes the details of the connections on the uInverter board. For detailed review of the connections, please refer [schematics](https://github.com/Severson-Group/AMDC-Hardware/tree/uinv-rev-a-pcb/Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a.pdf).
```

Comment:
Provide relative links -- this link won't work once we merge this PR.

Reply:
Done!